added batch mode to plugin install documentation (#920)
opensearch-project/documentation-website: _opensearch/install/plugins.md

Restart your OpenSearch node after installing a plugin.

## Batch mode

> When installing plugins that require additional privileges not included by default, the plugins will prompt the user for confirmation of the required privileges. The confirmation prompt can be skipped and grant all requested privileges for the plugins being installed by using batch mode.

Reviewer: ```suggestion
When installing plugins that require additional privileges not included by default, the plugins will prompt the user for confirmation of the required privileges. To grant all requested privileges, use batch mode to skip the confirmation prompt.
```
